src/common/NetworkManager.ts

```diff
@@ -368,6 +368,15 @@ export class NetworkManager extends EventEmitter {
       .catch(debugError);
   }
 
+  /**
+   * CDP may send a Fetch.requestPaused without or before a
+   * Network.requestWillBeSent
+   *
+   * CDP may send multiple Fetch.requestPaused
+   * for the same Network.requestWillBeSent.
```

Comment:
I would say all of this is already documented in detail at the start of the class definition